test/lib/prechallenge.js

```diff
@@ -0,0 +1,37 @@
+'use strict';
+
```

Comment:
This was such a great and inventive idea!

refactoring your tests into module helpers is a clean and great way to save time coding!
This looks really clean! :) 
Great work!